MoltenVK/MoltenVK/GPUObjects/MVKDeviceMemory.mm

```diff
@@ -44,14 +45,11 @@
 		return reportError(VK_ERROR_MEMORY_MAP_FAILED, "Memory is already mapped. Call vkUnmapMemory() first.");
 	}
```

Comment:
While I am not against these checks, it feels like they are redundant since VVL should catch those with `VUID-vkMapMemory-memory-00678` and `VUID-vkMapMemory-memory-00682`